praktikumsplaner-frontend/frontend/src/components/Praktikumsstellen/PraktikumsstelleCard.vue

```diff
@@ -173,6 +380,12 @@ function resetUnassign() {
     stelleToAssignUnassign.value = undefined;
     unassignConfirmDialog.value = false;
 }
+function noDropAllowed() {
+    useSnackbarStore().showMessage({
```

Comment:
When should this function be called?
I can't seem to get the message

Reply:
When entering the Praktikumsplatzoverview and trying to drop something onto a Praktikumsplatz